decidim-proposals/db/migrate/20240209092404_change_color_fields_on_proposals_states.rb

```diff
@@ -46,9 +42,5 @@ def up
     # rubocop:enable Rails/SkipsModelValidations
   end
 
-  def down
-    remove_column :decidim_proposals_proposal_states, :bg_color
-    remove_column :decidim_proposals_proposal_states, :text_color
-    add_column :decidim_proposals_proposal_states, :css_class, :string
-  end
+  def down; end
```

Comment:
We should raise an `IrreversibleMigration` here

Reply:
@andreslucena this handled in [7b36256](https://github.com/decidim/decidim/pull/13164/commits/7b362564fe49dd2ec93e465339324b40383bb69c)